app/Console/Commands/RetrieveRecordings.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Artist;
+use App\Models\Recording;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class RetrieveRecordings extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'recordings:daily';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle(): int
+    {
+        DB::statement("SET foreign_key_checks=0");
+        Recording::truncate();
+        Artist::truncate();
+        DB::statement("SET foreign_key_checks=1");
+        $opts = [
+            'http' => [
+                'method' => "GET",
+                'header' => "Accept-language: en\r\n" .
+                    "User-agent: Base ( dzibrovak@famcs-steps.yaconnect.com )\r\n"
+            ]
+        ];
+        $context = stream_context_create($opts);
+
+        $artists = json_decode(file_get_contents(
```

Comment:
Instead of using default php functions like file_get_contents its better to use Http clients like Guzzle 
https://laravel.com/docs/8.x/http-client